protocols/00f7b1_part10/README.md

```diff
@@ -0,0 +1,70 @@
+# NEBNext Ultra II Directional RNA Library Prep Kit for Illumina Part 10: Purification of PCR Product
+
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+
+## Categories
+* NGS LIBRARY PREP
```

Comment:
@ramifarawi @john-c-lynch can we also change the casing of this to `NGS Library Prep` - the current all caps creates a new category